engine/math/print_num.asm

```diff
@@ -6,7 +6,7 @@ _PrintNum::
 ; digits will be in front of the decimal point.
 ; Some extra flags can be given in bits 5-7 of b.
 ; Bit 5: money if set (unless left-aligned without leading zeros)
-; Bit 6: right-aligned if set
+; Bit 6: left-aligned if set
```

Comment:
Are you sure this even is right? Please verify.
If it is, also change the related `PRINTNUM_RIGHTALIGN(_F)` constants.

Reply:
Confirmed it to left-align.